tests/pipeline/test_adjusted_array.py

```diff
@@ -745,3 +745,66 @@ def test_update_labels(self):
         # values and adjustment values.
         check_arrays(adj_array.data, expected_adj_array.data)
         self.assertEqual(adj_array.adjustments, expected_adj_array.adjustments)
+
+    @parameterized.expand([(
+        # Initial adjustments
+        {
+            1: [Float64Multiply(0, 4, 1, 1, 0.5),
```

Comment:
One thing that might help clarify this further would be to bind these adjustments to variables, e.g.:

```
A = Float64Multiply(...)
B = Float64Overwrite(...)
```

Then your parameterized setup would look like:

```
# Initial
{1 : [A, B], 2: [C], 3: [D]},
# To add
{1: [E], 2: [F, G], 3: [H]},
# Expected with append.
...
# Expected with prepend.
```

The reason this might help is that the values in the adjustments themselves aren't really all that important here. What we care is just that, whatever the values in the adjustments, they get rearranged in the right way after merging.